src/oas2/transformers/params.ts

```diff
@@ -165,6 +165,7 @@ export function translateToQueryParameter(query: QueryParameter): IHttpQueryPara
 export function translateToPathParameter(parameter: PathParameter): IHttpPathParam {
   return (pickBy({
     ...buildSchemaForParameter(parameter),
+    description: parameter?.description,
```

Comment:
How is this different than the previous statement down on line 182?

Reply:
This resolves the following issue of inconsistent http http operation parameter description path: https://github.com/stoplightio/http-spec/issues/26